gips/data/sentinel2/sentinel2.py

```diff
@@ -1204,7 +1189,7 @@ def fmask_geoimage(self):
             "-a", "%s/allbands.vrt" % self._temp_proc_dir,
             "-z", "%s/angles.img" % self._temp_proc_dir,
             "-o", "%s/cloudmask.tif" % self._temp_proc_dir,
-            "--cloudprobthreshold", "5",
+            "--cloudprobthreshold", "22.5",
```

Comment:
hrm...out of scope for this PR, but we should review the consequences of this change.  perhaps push it out to a `sentinel2Data.extra_args` cli parameter (which involves resurrecting the feature in the style of `Data.add_filter_args`).

Reply:
@bhbraswell ^^^^^ (look up here)